File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int a;

    do
    {
     a = get_int("Height: ");
    }
    while (a > 8 || a < 1);
// now to write actual # rows

  for (int i=0; i<a ; i++)
  {
    for (int j= a-i-1; j>0; j--)
      {
        printf(".");
      }
    for (int k= 0)
      printf("\n");



  }


}

File: sentimental/mario-less/mario.py
from cs50 import get_int

def inp():
    height = get_int("Height: ")
    main(height)

def main(n):

    if n > 8 or n < 1 :
        inp()

    else :

        for i in range(0,n,1):
            print(f"{i}")



main()

